README.md

```diff
@@ -42,6 +43,11 @@ fully functional electrode localization pipeline.
 
 ## Usage
 
+### Video Tutorial
+
+[![DataJoint Element Video Tutorial](https://img.youtube.com/vi/YRXokFHkLGg/0.jpg)](https://www.youtube.com/watch?v=YRXokFHkLGg)
```

Comment:
The screenshot doesn't render great for me - I don't think the screenshare video style lends itself to great thumbnails. I think I would keep our norm of text links until we can get consistent support in generating thumbnails 
![image](https://user-images.githubusercontent.com/9404707/167174586-996100ef-5ea7-4bf0-87d2-24c964bfdb8d.png)


```suggestion
[Element Video Tutorial](https://www.youtube.com/watch?v=YRXokFHkLGg)
```

Reply:
Thanks @CBroz1.  I have added a thumbnail to this video.

![Thumbnails](https://user-images.githubusercontent.com/871137/167220078-6e6b172c-d70c-45bf-8cf3-20f8fb5a67d7.png)